automation/taskcluster/helper/get-secret.py

```diff
@@ -20,7 +22,17 @@ def write_secret_to_file(path, data, key, base64decode=False, json_secret=False,
 
 
 def fetch_secret_from_taskcluster(name):
-    secrets = taskcluster.Secrets({'baseUrl': 'http://taskcluster/secrets/v1'})
+    try:
+        secrets = taskcluster.Secrets({
+            # BaseUrl is still needed for tasks that haven't migrated to taskgraph yet.
+            'baseUrl': 'http://taskcluster/secrets/v1',
+        })
+    except taskcluster.exceptions.TaskclusterFailure:
+        # taskcluster library >=5 errors out when `baseUrl` is used
```

Comment:
Oh, while we're in this migration period, are we using different versions of the `taskcluster` library (both `4` and `5`)?

Reply:
Yes. The new docker image installs the latest `taskcluster` lib, while the old manually crafted images are stuck on v4.